src/video-grid/useMediaStream.ts

```diff
@@ -35,21 +35,31 @@ declare global {
 export const useMediaStreamTrackCount = (
```

Comment:
I am confused about what the changes here are doing

Reply:
I think the type of `stream` should be changed to reflect that then (or the other option is splitting `AudioForParticipant` into an outer and inner component to avoid adding nullability)

Reply:
Done -  I was planning on going through & doing a strict-mode sweep soon so was sort of neglecting these.